tests/conftest.py

```diff
@@ -3,16 +3,23 @@
 This is a PyTest special file, see its documentation.
 """
 import logging
+from threading import Thread
 from unittest.mock import patch
+from wsgiref.simple_server import WSGIRequestHandler, WSGIServer
 
 from cryptography.hazmat.primitives.asymmetric import rsa
 from cryptography.hazmat.backends import default_backend
 import pytest
 
+# TODO: the below hack will go when the complete API is implemented,
```

Comment:
Ah, yes, but we're still registering assets with the registry, and that doesn't go through the REST API. Once the new identifiers are there (https://github.com/SecConNet/proof_of_concept/issues/18) it can go, but not yet...

Reply:
Check